iceberg/iceberg-common/src/main/java/org/apache/gravitino/iceberg/common/ops/IcebergCatalogWrapper.java

```diff
@@ -187,12 +208,19 @@ public ListTablesResponse listTable(Namespace namespace) {
   }
 
   public void renameTable(RenameTableRequest renameTableRequest) {
+    metadataCache.invalidate(renameTableRequest.source());
```

Comment:
Should we also invalidate the dest name?

Reply:
seems not necessary.  if `rename` succ,  dest table metadata should not in the cache.  If `rename` failed, should not remove dest table cache too.